notebooks/benchmark/Default Mask Benchmark.ipynb

```diff
@@ -0,0 +1,283 @@
+{
```

Comment:
it would be nice to have both documentation in the notebooks and in the folder in general

Reply:
I'll add some more comments in the notebooks and the folder to help clarify the steps.